pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -438,6 +453,10 @@ func (s *nsmgrSuite) Test_PassThroughRemoteUsecase() {
 	// Path length from NSE client to other remote endpoint is 8
 	require.Equal(t, 8*(nodesCount-1)+5, len(conn.Path.PathSegments))
```

Comment:
Probably it would be better to check that we have visited all required endpoints only once in the required order.

Reply:
done